src/keys.c

```diff
@@ -1226,9 +1226,41 @@ int SetCipherSpecs(WOLFSSL* ssl)
         }
     }
 
+    if (ssl->options.cipherSuite0 == ECDHE_PSK_BYTE) {
```

Comment:
Should this be wrapped with `#ifndef NO_PSK`?

Reply:
None of the ones above are wrapped in `NO_PSK` and `BUILD_TLS_ECDHE_PSK_WITH_AES_128_GCM_SHA256` depends on PSK.